OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -127,7 +127,7 @@ public WVec GetRepulsionForce(Actor other)
 
 		public Actor GetActorBelow()
 		{
-			if (self.CenterPosition.Z != 0)
+			if (self.World.Map.DistanceAboveTerrain(CenterPosition).Length != 0)
```

Comment:
@Phrohdoh this is for when an aircraft is docked. A trait similar to `Aircraft` could be used to go underground in which case you would what `GetActorBelow()` to return `null` as well if not at ground level.